server/embedded/build.gradle

```diff
@@ -31,6 +31,9 @@ dependencies {
     implementation "org.springframework.boot:spring-boot-starter-web:${springBootVersion}"
     implementation "org.springframework.boot:spring-boot-starter-actuator:${springBootVersion}"
     implementation "org.springframework.boot:spring-boot-starter-validation:${springBootVersion}"
+    
+    // Specify a version to get the latest hotfix for a CVE. Can revert once Spring Boot updates
+    implementation "org.springframework:spring-expression:${springVersion}"
```

Comment:
@labkey-susanh this worked and seemed a little leaner than the `strictly` we had to use for `snakeyaml`. Let me know if you prefer a different syntax.